app/stories/components/ReviewCollection.stories.tsx

```diff
@@ -5,6 +5,11 @@ import ReviewCollectionComponent from '@components/ReviewCollection'
 export default {
   title: 'Components',
   component: ReviewCollectionComponent,
+  argTypes: {
+    editHref: {
+      control: 'text',
+    },
+  },
```

Comment:
Notes for reviewers: Show the `editHref` arg as a text input control rather than a json control.

Reply:
Is this better as a text control? I'm not sure what the difference is 😅